### PR TITLE
Replace **+ New** button content

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Replace **+ New** button, which is deprecated in the New UI, with alternate **Home** dashboard instructions.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-8

### CODE REVIEW

File: docs/apm/real-user-monitoring/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ See Real User Monitoring in action.
 
 Access Traces to confirm that your Sumo Logic service package has been upgraded to include Traces and Real User Monitoring.
 
-[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, click the **+ New** button at the top of the screen and select **Traces**.
+[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, go to your **Home** dashboard and select **Traces**.
```
Comment (jpipkin1) [inline docs/apm/real-user-monitoring/index.md:50]:
```suggestion
[**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access Traces, go to the **Home** screen and select **Traces**.
```

Comment (jpipkin1) [reply docs/apm/real-user-monitoring/index.md:50]: I suggest globally changing "your **Home** dashboard" to "the **Home** screen". "Dashboard" could be confused with the dashboards functionality, and "your" implies it's a screen specific to the user.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Thanks for making this change, @kimsauce! I have just one suggestion.

File: docs/apm/real-user-monitoring/index.md (hunk 1 above)
Comment (kimsauce) [reply docs/apm/real-user-monitoring/index.md:50]: Thank you! Much better. I'll make this change globally.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce! I made one tiny fix to that global change (see commit "[Minor update from review](https://github.com/SumoLogic/sumologic-documentation/pull/4582/commits/e6d36c7f7f02266d817456c92a6ca87fbcd0563a)").

**Comment by kimsauce** [comment]
> Looks good, @kimsauce! I made one tiny fix to that global change (see commit "[Minor update from review](https://github.com/SumoLogic/sumologic-documentation/pull/4582/commits/e6d36c7f7f02266d817456c92a6ca87fbcd0563a)").

Gahhh, I am off my game today 😂 Thank you so much for your thorough review/catches, @jpipkin1 !

**Pull request merged**